dev/table_name_collector.h

```diff
@@ -33,19 +37,19 @@ namespace sqlite_orm {
             }
 
             template<class T, class F>
-            void operator()(const column_pointer<T, F>&) const {
+            void operator()(const column_pointer<T, F> &) const {
```

Comment:
these fixes do not exist in `sqlite_orm.h`. Please rerun `amalgamate.py` script and push it